x-pack/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -400,6 +397,20 @@ export function getBoundingBoxGeometry(geometry) {
   return formatEnvelopeAsPolygon(extent);
 }
 
+export function makeGeotileGridDsl(field, bounds, precision) {
```

Comment:
No need for both makeGeotileGridDsl and makeSizedGeotileGridDsl. Just make a single method.

```
export function makeGeotileGridDsl(field, bounds, precision, aggParams = {}) {
  const esBbox = makeESBbox(bounds);
  return {
    bounds: makeESBbox(bounds),
    field,
    precision,
    ...aggParams,
  };
}

```